packages/rule-tester/src/utils/flat-config-schema.ts

```diff
@@ -10,17 +10,17 @@ import { normalizeSeverityToNumber } from './severity';
 type PluginMemberName = `${string}/${string}`;
 
 interface ObjectPropertySchema<T = unknown> {
-  merge: string | ((a: T, b: T) => T);
-  validate: string | ((value: unknown) => asserts value is T);
+  merge: ((a: T, b: T) => T) | string;
+  validate: ((value: unknown) => asserts value is T) | string;
 }
 
 const ruleSeverities = new Map<SharedConfig.RuleLevel, SharedConfig.Severity>([
-  [0, 0],
+  ['error', 2],
   ['off', 0],
-  [1, 1],
   ['warn', 1],
+  [0, 0],
+  [1, 1],
   [2, 2],
```

Comment:
this is another case where the value ordering is better than the prop name ordering